ReactiveCocoa/Swift/ObjectiveCBridging.swift

```diff
@@ -15,15 +15,15 @@ extension RACScheduler: DateSchedulerType {
 	}
 
 	public func schedule(action: () -> ()) -> Disposable? {
-		return self.schedule(action)
+		return schedule(action)
```

Comment:
I'd rather keep the `self`: it makes it clear that this is an instance property instead of a global, local variable or parameter.